src/shared/utils/tooltip.ts

```diff
@@ -163,18 +166,22 @@ export const showTooltipAtCoordinates = (
   document.body.addEventListener('scroll', hideTooltip, true);
   document.body.addEventListener('wheel', hideTooltip, true);
 
-  function hideTooltip() {
-    tooltip?.remove();
-    cleanup?.();
-    document.body.removeEventListener('click', onClick, true);
-    document.body.removeEventListener('scroll', hideTooltip, true);
-    document.body.removeEventListener('wheel', hideTooltip, true);
+  function hideTooltip(e?: Event) {
+    const target = e?.target as Node;
+    // f scroll/wheel is within the tooltip element, do not remove the tooltip
+    if (target && !tooltip.contains(target)) {
```

Comment:
This seems duplicated by L183?

Reply:
that' right. There is no need for two separate functions. hideTooltip is complete. Do you have any suggestion for a name of the event handler?